pkg/machines/helpers.es6

```diff
@@ -108,7 +108,18 @@ const transform = {
         crashed: _("crashed"),
         dying: _("dying"),
         pmsuspended: _("suspended (PM)"),
-    }
+    },
+    'bootableDisk': {
+        disk: _("disk"),
+        cdrom: _("cdrom"),
+        'interface': _("network"),
```

Comment:
The others are not needed, but the 'interface' is reserved word.
For consistency, I'm adding quotes for the remaining as well.